sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway.bpmn

```diff
@@ -789,10 +568,342 @@
+        <zeebe:taskDefinition type="associate-workflow-instance" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1ef3map</bpmn:incoming>
+      <bpmn:outgoing>Flow_1wrj7fh</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:startEvent id="assessment-start-event">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=null" target="applicationId" />
+          <zeebe:output source="=assessmentId" target="assessmentId" />
+          <zeebe:output source="=null" target="applicationExceptionStatus" />
+          <zeebe:output source="=null" target="studentDataSelectedOffering" />
+          <zeebe:output source="=null" target="studentDataDependantstatus" />
+          <zeebe:output source="=null" target="studentDataParentValidSinNumber" />
+          <zeebe:output source="=null" target="studentDataNumberOfParents" />
+          <zeebe:output source="=null" target="studentDataRelationshipStatus" />
+          <zeebe:output source="=null" target="studentDataIsYourPartnerAbleToReport" />
+          <zeebe:output source="=null" target="parent1TotalIncome" />
+          <zeebe:output source="=null" target="parent2TotalIncome" />
+          <zeebe:output source="=null" target="partner1TotalIncome" />
+          <zeebe:output source="=null" target="studentDataTaxReturnIncome" />
+          <zeebe:output source="=null" target="studentDataCurrentYearIncome" />
+          <zeebe:output source="=[]" target="parentSubprocesses" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:outgoing>Flow_1ef3map</bpmn:outgoing>
+    </bpmn:startEvent>
+    <bpmn:callActivity id="load-assessment-data-submit-or-reassessment-subprocess" name="Load consolidated data (Submit or Reassessment)">
+      <bpmn:extensionElements>
+        <zeebe:calledElement processId="load-assessment-consolidated-data" propagateAllChildVariables="true" />
+        <zeebe:ioMapping>
+          <zeebe:input source="=[&#34;load-assessment-data-submit-or-reassessment-subprocess&#34;]" target="parentSubprocesses" />
+          <zeebe:input source="=&#34;load-assessment-data-submit-or-reassessment-subprocess&#34;" target="load_assessment_data_submit_or_reassessment_subprocess_passthrough" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1wrj7fh</bpmn:incoming>
+      <bpmn:outgoing>Flow_0rxxozm</bpmn:outgoing>
+    </bpmn:callActivity>
+    <bpmn:sequenceFlow id="Flow_1ef3map" sourceRef="assessment-start-event" targetRef="associate-workflow-instance-task" />
+    <bpmn:sequenceFlow id="Flow_1wrj7fh" sourceRef="associate-workflow-instance-task" targetRef="load-assessment-data-submit-or-reassessment-subprocess" />
+    <bpmn:sequenceFlow id="Flow_0g0yluv" name="Original application submission" sourceRef="Gateway_1imkna6" targetRef="Gateway_139oo1r">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=applicationEditStatus = "Original"</bpmn:conditionExpression>
```

Comment:
What about `applicationEditStatus = "Edited"` ? The edit before COE.

Reply:
Edit before COE never reaches the workflow as 'Edited', it follows a new submission from scratch.

Reply:
I raised the question looking at this block of code where new application created has the edit_status as `Edited`.  Let me know if I am missing something here.

![image](https://github.com/user-attachments/assets/c7571a0a-c9cc-444a-846c-505d19d759c4)

Reply:
Got it now. I am changing that flow to be the default one from now on.